packages/gatsby/src/pages/configuration/yarnrc.json

```diff
@@ -24,6 +24,14 @@
       },
       "default": ["master", "origin/master", "upstream/master"]
     },
+    "changesetIgnorePatterns": {
+      "description": "Files matching the following patterns (in terms of relative paths compared to the root of the repository) will be ignored by the `yarn version check` command.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "default": ["**/*.test.{js,ts}"]
```

Comment:
I know, but I never considered the `default` from the JSON schemas to be the actual default, but just an example. It's the same with options like `unsafeHttpWhitelist` or `packageExtensions`. I just needed a property in the JSON schemas to generate the example from. This could be changed in the future when I finally decide to refine the config from JSON schemas generation process.